### DISCUSSION TITLE
KYK internetinde çalışmıyor

### DISCUSSION THREAD

**Discussion by 10urdev** [cagritaskn/GoodbyeDPI-Turkey#19] [Q&A] [answered]
KYK interneti olan GSBWifi de bizim yurdumuzda geçen sene tüm oyunlar çalışıyorken bu sene herhangi bir oyuna girilmiyor. (Bu arada discordda açılmıyor) Tam çalışma mantığını bilmiyorum ancak bu sorun goodbyeDPI ile aşılabilir mi? İnternet ağlarının çalışma mantığına hakim olmadığım için kodlardan da pek bir şey anlayamıyorum. Eğer böyle bir imkan varsa ve yapabilirseniz ya da yapmak için gerekli adımları söyleyebilirseniz ilk başta ben olmak üzere çoğu öğrenci size teşekkür edecektir. İyi günler dilerim

**Comment by cagritaskn** [comment, marked as answer]
Hocam öncelikle KYK Wi-Fi'ına bağlanmak için bir giriş sitesi olduğunu biliyorum. Bu sitenin açılması için GoodbyeDPI'ın kapalı kalması gerekiyor. KYK Wi-Fi'ına giriş bilgilerinizi kullanarak bağlandıktan sonra GoodbyeDPI'ı çalıştırarak işinize yarayıp yaramadığını test edebilirsiniz.

**Reply by 10urdev** [reply]
Hayır her şekilde denedim çalışmıyor maalesef :/
